06_neighborhoods/Transportation/transit_trips_county.R

```diff
@@ -0,0 +1,203 @@
+  )
+
+# keep only variables of interest
+transit_trips_county_2019 <- transport_county_2019 %>%
+  select(state, county, blkgrps, population, households, transit_trips_80ami)
+
+# Compare to our official county file to make sure we have all counties accounted for
+# Bring in the official county file
+counties <- read_csv("C:/Users/USERNAME/Documents/GitHub/mobility-from-poverty/geographic-crosswalks/data/county-populations.csv")
+counties_2015 <- counties %>%
+  filter(year == 2015)
+counties_2019 <- counties %>%
+  filter(year == 2019)
+# all files have same number of observations (3142) so no merging needed to account for missings!
+
+
+###################################################################
+
+# 2. QC Checks
+# create a histogram plot and summary stats for each dataframe
+# check that all values are non-negative & count missing values
+# examine outliers
+
+# County-Level Transit Trips 2015
+ggplot(transit_trips_county_2015, aes(x=transit_trips_80ami)) + geom_histogram(binwidth=5) + labs(y="number of counties", x="Annual Transit Trips for the Regional Moderate Income Household, 2015")
+# Makes sense for most counties to fall in really low transit trip numbers since most of the US has no public infrastructure that can be used for daily transport
+# look at summary stats
+summary(transit_trips_county_2015$transit_trips_80ami)
+# examine outliers
+transit_trips_county_2015_outliers <- transit_trips_county_2015 %>% 
+  filter(transit_trips_80ami>250) 
+# 1057 trips in 38059 county -- Morton County, North Dakota -- seems to be a railroad town...
+# 1300 trips in 36047 -- Kings County, New York -- this is Brooklyn, makes total sense
+# all the highest ones(1000+) are in New York counties - checks out
+# 313 trips in 06037 -- Los Angeles county, California - checks out
+
+# Use stopifnot to check if all values in "transit_trips_county_2015" are non-negative
+stopifnot(min(transit_trips_county_2015$transit_trips_80ami, na.rm = TRUE) >= 0)
+# Good to go
+
+# Find indices of missing values for the "transit_trips_80ami" variable
+missing_indices <- which(is.na(transit_trips_county_2015$transit_trips_80ami))
+# Print observations with missing values
+if (length(missing_indices) > 0) {
+  cat("Observations with missing values for transit_trips_80ami:\n")
+  print(transit_trips_county_2015[missing_indices, , drop = FALSE])
+} else {
+  cat("No missing values for transit_trips_county_2015.\n")
+}
+# One missing value: Loving County, Texas (FIPS 48301)
+
+
+# County-Level Transit Trips 2019
+ggplot(transit_trips_county_2019, aes(x=transit_trips_80ami)) + geom_histogram(binwidth=5) + labs(y="number of counties", x="Annual Transit Trips for the Regional Moderate Income Household, 2019")
+# looks similar, checks out
+# look at summary stats
+summary(transit_trips_county_2019$transit_trips_80ami)  
+# examine outliers
+transit_trips_county_2019_outliers <- transit_trips_county_2019 %>% 
+  filter(transit_trips_80ami>250) 
```

Comment:
If you are creating a dataframe for manual outlier checking it is helpful to sort it largest to smallest.